client/app/billing/billing.routing.js

```diff
@@ -0,0 +1,25 @@
+import template from './billing.html';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.account.billing', {
+    url: '/billing',
+    // needs to be imported
+    abstract: true,
+    translations: { value: ['.'], format: 'json' },
+    template,
+    controller: 'BillingCtrl',
+    resolve: {
+      denyEnterprise: ($q, currentUser) => {
+        if (currentUser.isEnterprise) {
+          return $q.reject({
+            status: 403,
+            message: 'Access forbidden for enterprise accounts',
+            code: 'FORBIDDEN_BILLING_ACCESS',
+          });
+        }
+
+        return true;
```

Comment:
Shouldn't it be `false` ? If `current.isEnterprise` === false, then we don't deny entreprise ?

Reply:
I suppose this was to differentiate the Promise rejection from else but sounds more logical indeed :)